File: composeApp/src/androidMain/kotlin/authentication/HeaderSection.kt
package authentication

import androidx.compose.foundation.Image
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.layout.size
import androidx.compose.material3.Icon
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.SegmentedButtonDefaults.Icon
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.unit.dp
import com.example.multimodule.R

@Composable
fun HeaderSection() {
    Column(horizontalAlignment = Alignment.CenterHorizontally) {
        // Replace with actual illustration/icon
        Image(
            painter = painterResource(id = R.drawable.bitcoin),
            contentDescription = "app Image",
            modifier = Modifier.size(80.dp)
        )
        Text(
            text = "Your Family’s Healthcare in One App",
            style = MaterialTheme.typography.headlineSmall,
            textAlign = TextAlign.Center
        )
        Text(
            text = "Manage appointments, access records, and consult doctors securely.",
            style = MaterialTheme.typography.bodyMedium,
            textAlign = TextAlign.Center,
            modifier = Modifier.padding(top = 8.dp)
        )
    }
}

File: iosApp/iosApp/uiComponents/authentication/HeaderSection.swift
import SwiftUI

struct HeaderSection: View {
    var body: some View {
        VStack(spacing: 8) {
            Image(systemName: "heart.text.square.fill")
                .resizable()
                .frame(width: 80, height: 80)
                .foregroundColor(.blue)
            Text("Your Family’s Healthcare in One App")
                .font(.title2)
                .fontWeight(.semibold)
                .multilineTextAlignment(.center)
            Text("Manage appointments, access records, and consult doctors securely.")
                .font(.subheadline)
                .foregroundColor(.secondary)
                .multilineTextAlignment(.center)
        }
    }
}